app/jobs/regular/chat_channel_archive.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Jobs
+  class ChatChannelArchive < ::Jobs::Base
+    sidekiq_options retry: false
+
+    def execute(args = {})
+      channel_archive = ::ChatChannelArchive.find_by(id: args[:chat_channel_archive_id])
+      return if channel_archive.blank?
```

Comment:
When ChatChannelArchive can be blank? If the record is blank, doesn't it mean that something bad happened and here we are suppressing that?

Reply:
Yeah, I will throw a warning in there, shouldn't really happen but better than throwing errors in the job :)